datadog_checks_base/datadog_checks/base/checks/win/winpdh.py

```diff
@@ -19,10 +19,11 @@ class WinPDHCounter(object):
     pdh_counter_dict = defaultdict(list)
     _use_en_counter_names = False
 
-    def __init__(self, class_name, counter_name, log, instance_name=None, machine_name=None, precision=None):
+    def __init__(self, en_class_name, en_counter_name, log, instance_name=None, machine_name=None, precision=None):
         self.counterdict = {}
         self.logger = log
-        self._counter_name = counter_name
+        self._counter_name = en_counter_name
```

Comment:
Is setting `_counter_name = en_counter_name` to give a default value for `_counter_name`? If so, why don't you also do this for `_class_name`?

Reply:
That's how the rest of the code works. The stored value for `counter_name` is always the english version. The stored value for the `class_name` is always the localized version.
This is also done for `_class_name` a few lines after